src/Razor/test/Microsoft.AspNetCore.Razor.LanguageServer.Test/Formatting/HtmlFormattingTest.cs

```diff
@@ -767,6 +767,19 @@ public async Task FormatNestedComponents3()
                                 </Frag>
                                 </Component1>
                     }
+
+                    @if (true)
+                    {
+                        <a_really_long_tag_name Id="comp1"
```

Comment:
Added this because html tags formatted correctly, and component tags didn't, so it was convenient when debugging to see what was happening that was different. Figured I'd leave it 🤷‍♂️